src/woo_publications/templates/admin/change_owner.html

```diff
@@ -0,0 +1,91 @@
+{% extends "admin/delete_selected_confirmation.html" %}
+{% load i18n l10n admin_urls static %}
+
+{% block extrahead %}
+    {{ block.super }}
+    {{ media }}
+    <script src="{% static 'admin/js/cancel.js' %}" async></script>
+    <link rel="stylesheet" type="text/css" href="{% static 'admin/css/widgets.css' %}">
+    <link rel="stylesheet" type="text/css" href="{% static 'admin/css/forms.css' %}">
+    <link rel="stylesheet" type="text/css" href="{% static 'admin/css/admin-index.css' %}">
+{% endblock %}
+
+{% block bodyclass %}{{ block.super }} app-{{ opts.app_label }} model-{{ opts.model_name }} change-owner-confirmation change-owner-selected-confirmation{% endblock %}
+
+{% block breadcrumbs %}
+<div class="breadcrumbs">
+<a href="{% url 'admin:index' %}">{% translate 'Home' %}</a>
+&rsaquo; <a href="{% url 'admin:app_list' app_label=opts.app_label %}">{{ opts.app_config.verbose_name }}</a>
+&rsaquo; <a href="{% url opts|admin_urlname:'changelist' %}">{{ opts.verbose_name_plural|capfirst }}</a>
+&rsaquo; {% translate 'Change owner of multiple objects' %}
+</div>
+{% endblock %}
+
+{% block content %}
+    <fieldset class="module aligned ">
+        <p>{% blocktranslate %}Are you sure you want to change the owner of the selected {{ objects_name }}? All of the following objects will have their owner changed:{% endblocktranslate %}</p>
+        <ul>
+            {% for changeable_object in changeable_objects %}
+                <li>{{ changeable_object }}</li>
+            {% endfor %}
+        </ul>
+        <hr />
+    </fieldset>
+
+    <fieldset class="module aligned ">
+        <p>
+        {% blocktranslate %}
```

Comment:
```suggestion
        {% blocktranslate trimmed %}
```
should be the default IMO, this makes the translations easier to manage in rosetta/po files